ustaxcourt/ef-cms
# Story 9464: Multiple Clinic Letters for Different Addresses

Addressed the requirements in the story, but also...

- Tests
  - Make a new describe block for all the clinic letter tests.
  - Remove the second petitioner from the existing clinic letter tests because they will be covered in our new tests.
  - Added new tests for multiple petitioners based on the tests in the story.
- Actual code
  - Made a dedicated function that computes when a clinic letter is needed since it is complicated.
  - Moved the NOTR generation for the second petitioner out of a helper function and converted it to just check if the addresses are different.
  - Use the combine PDF utility more.
